packages/components/src/internal/components/administration/actions.ts

```diff
@@ -83,3 +88,67 @@ export function updateSecurityPolicy(
         });
     });
 }
+
```

Comment:
Not totally sure if these two functions are located in a sensible place.

Reply:
this administration/actions.ts file seems reasonable

Reply:
should the function be renamed to `getGroupMembers` or something along those lines?